concrete_datastore/api/v1/authentication.py

```diff
@@ -56,18 +57,17 @@ def api_token_has_expired(token):
     return False
 
 
-def expire_secure_token(token):
+def expire_secure_connect_instance(instance, expiration_limit_in_seconds):
```

Comment:
function name is not explicit enough, since it does not expire the "connect_instance" if the expiration_date has not passed

Reply:
The function is renamed `ensure_secure_connect_instance_is_not_expired` and the logic inverted